files/sidebars/htmlsidebar.yaml

```
@@ -3,221 +3,91 @@
-        title: Block-level_elements
-      - link: /Glossary/Inline-level_content
-        title: Inline_elements
-      - link: /Web/HTML/Quirks_Mode_and_Standards_Mode
-        title: Quirks_Mode_and_Standards_Mode
-      - link: /Web/HTML/Date_and_time_formats
-        title: Date_and_time_formats
-      - link: /Web/HTML/Constraint_validation
-        title: Constraint_validation
-      - link: /Web/HTML/Microdata
-        title: Microdata
-      - link: /Web/HTML/microformats
-        title: Microformats
-      - link: /Web/HTML/Viewport_meta_tag
-        title: Viewport_meta_tag
-      - link: /Web/HTML/CORS_enabled_image
-        title: Allowing_cross-origin_images_and_canvas
+  - type: section
+    link: /Web/HTML/Guides
+    title: Guides
+  - type: listSubPages
+    path: /Web/HTML/Guides
+  - type: section
+    link: /Web/HTML/How_to
+  - type: listSubPages
+    path: /Web/HTML/How_to
   - type: section
     link: /Web/HTML/Reference
     title: Reference
   - type: listSubPages
-    path: /Web/HTML/Element
+    path: /Web/HTML/Reference/Elements
     title: HTML_Elements
     details: closed
     code: true
   - type: listSubPages
-    path: /Web/HTML/Global_attributes
-    title: Global_attributes
+    path: /Web/HTML/Reference/Elements/input
+    title: Input_types
     details: closed
     code: true
   - type: listSubPages
-    path: /Web/HTML/Attributes
+    path: /Web/HTML/Reference/Attributes
     title: Attributes
     details: closed
   - type: listSubPages
-    path: /Web/HTML/Element/input
-    title: Input_types
+    path: /Web/HTML/Reference/Global_attributes
+    title: Global_attributes
     details: closed
     code: true
 l10n:
   de:
-    Beginners: Einsteiger-Tutorials
-    Creating_the_content: "Ihre erste Website: Inhalte erstellen"
-    Structuring_content_with_HTML: Inhalte mit HTML strukturieren
     Reference: Referenzen
     HTML_Elements: HTML-Elemente
     Global_attributes: Globale Attribute
     Attributes: Attribute
     Input_types: <code>&lt;input&gt;</code>-Typen
     Guides: Anleitungen
-    Content_categories: Inhaltskategorien
-    Block-level_elements: Block-Elemente
-    Inline_elements: Inline-Elemente
-    Quirks_Mode_and_Standards_Mode: Quirks-Modus und Standards-Modus
-    Date_and_time_formats: Datums- und Zeitformate in HTML
-    Constraint_validation: Einschränkungsvalidierung
-    Microdata: Mikrodaten
-    Microformats: Mikroformate
-    Viewport_meta_tag: Viewport-Meta-Tag
-    Allowing_cross-origin_images_and_canvas: Erlauben der Cross-Origin-Nutzung von Bildern und Canvas
   en-US:
-    Beginners: Beginner's tutorials
-    Creating_the_content: "Your first website: Creating the content"
-    Structuring_content_with_HTML: Structuring content with HTML
-    Reference: References
-    HTML_Elements: HTML elements
+    Reference: Reference
+    HTML_Elements: Elements
     Global_attributes: Global attributes
     Attributes: Attributes
     Input_types: <code>&lt;input&gt;</code> types
```

If we do Script Types:

```suggestion
    Input_types: <code>&lt;input&gt;</code> types
    Script_types: <code>&lt;script&gt;</code> types
```

---

Yeah, let's get some platform support for this. There's some sub-pages in Attributes as well that would be nice to show up in the side bar